### DISCUSSION TITLE
Welcome to Final_Project_KEL8 Discussions!

### DISCUSSION THREAD

**Discussion by BIMA-25** [BIMA-25/Final_Project_KEL8#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BIMA-25** [comment]
halo guys untuk kedepanya dalam kita berdiskusi kita dapat gunain ftur diskusi ini ya

**Comment by afifahfau246** [comment]
Gais mau nanya, aku belum paham bagian kirim notif ke telegram itu yang integrasinya langsung dari alat, kalau boleh tolong dijelasin

**Comment by afifahfau246** [comment]
Maksud koding dari alatnya itu gimana?

**Comment by BIMA-25** [comment]
langsung hard code dari alatnya fah
